docs/messaging/messaging-spans.md

```diff
@@ -202,40 +194,30 @@ A producer of a message should set the span kind to `PRODUCER` unless it synchro
 The processor of the message should set the kind to `CONSUMER`, unless it always sends back a reply that is directed to the producer of the message
 (as opposed to e.g., a queue on which the producer happens to listen): then it should use `SERVER`.
 
-### Operation names
-
-The following operations related to messages are defined for these semantic conventions:
-
-| Operation name | Description |
-| -------------- | ----------- |
-| `publish`      | A message is sent to a destination by a message producer/client.       |
-| `receive`      | A message is received from a destination by a message consumer/server. |
-| `process`      | A message that was previously received from a destination is processed by a message consumer/server. |
```

Comment:
Let's please keep this as a dedicated section.

There are changes proposed in #284 which restructures the document a bit so that some following sections ("Span kind" and "Trace structure") depend on a definition of those operations.